index.bs

```diff
@@ -542,6 +542,15 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 			* Custom format [=string/starts with=] `"web "`("web" followed by U+0020 SPACE) prefix
 				and suffix (after stripping out `"web "`) passes the [=parsing a MIME type=] check.
 
+	<h3 id="unsanitized-data-types-x"><dfn>Unsanitized data types</dfn></h3>
+
+			These data types MUST NOT be sanitized by UAs:
+
+			* image/png
+
+			<p class=note>
+				The image/png MIME type is not sanitized to preserve meta data.
```

Comment:
This comment still doesn't really make it clear to me why we don't sanitize images. The statement "preserve meta data" is far too high level. Can we add more details here, or even better, an example?

Reply:
This was added as part of shipping the `image/png` MIME type in Chromium. I'm not very familiar with what metadata was getting stripped out due to sanitization, so I'll have to talk to the owner to understand all the details.

Reply:
In this case, I would just leave out the note entirely and just leave it as "image/png". It would still be worth understanding this aspect better and then adding some text in the spec as explanation as a follow up.

Reply:
Done.